### DISCUSSION TITLE
Java｜如何用一个统一结构接收成员名称不固定的数据 — 码志

### DISCUSSION THREAD

**Discussion by giscus** [mzlogin/blog-comments#277] [Announcements]
# Java｜如何用一个统一结构接收成员名称不固定的数据 — 码志

本文介绍了一种 Java 中如何用一个统一结构接收成员名称不固定的数据的方法。

https://mazhuang.org/2024/11/29/java-structure-with-uncertain-member-names/

**Comment by ghost** [comment]
hello
